### DISCUSSION TITLE
Internal link error when using minio s3 with same host url

### DISCUSSION THREAD

**Discussion by warnus** [outline/outline#3274] [Self Hosting]
Hello,

I met internal link(document link) error.
Whe I write another document link in current document, it changed attachment link.

Because I'm using minio S3 in same server. So, the AWS_S3_UPLOAD_BUCKET_URL is same with wiki HOST URL.
Maybe that is the problem.

Does anybody knows how to fix it?

Thank you.

**Comment by tommoor** [comment]
What is an "attachment link", please provide screenshots of the problem if possible

**Comment by warnus** [comment]
Thanks @tommoor 
I meant "attachment component" like this. Sorry for the confusion. 
<img width="410" alt="wiki_error" src="https://user-images.githubusercontent.com/5237424/159208545-32c667da-e86e-4867-8b07-e56fc79d5daf.png">

I put another document link in current edit document. And then, when I publish document, the link changed attachement component.

For example,
https://wiki.example.com/doc/642w7j207yswioybjo2bro2ujouhnoyasa-9OJEFkwHbt

It changed Attachment Component as 
<img width="410" alt="wiki_error" src="https://user-images.githubusercontent.com/5237424/159208545-32c667da-e86e-4867-8b07-e56fc79d5daf.png">

**Reply by tommoor** [reply]
Ah weird, file attachments isn't in an official build yet though so some bugs are expected. Thanks for raising, it'll need code changes to fix. You can avoid by using the official docker build: `outlinewiki/outline:latest`

**Reply by warnus** [reply]
Thanks a lot. @tommoor
